CMakeLists.txt

```diff
@@ -199,25 +199,46 @@ install(FILES ${CMAKE_SOURCE_DIR}/share/icons/chewing-editor.svg DESTINATION ${C
 
 # manpage
 find_program(HELP2MAN help2man)
+find_program(GZIP gzip)
 
 if (HELP2MAN)
-    set(manpage ${PROJECT_BINARY_DIR}/chewing-editor.1.gz)
     set(h2m ${PROJECT_SOURCE_DIR}/chewing-editor.h2m)
 
-    add_custom_command(
-        OUTPUT
-            ${manpage}
-        COMMAND ${HELP2MAN}
-            --include=${h2m}
-            --section=1
-            --no-info
-            --no-discard-stderr
-            --output=${manpage}
-            ${PROJECT_BINARY_DIR}/chewing-editor
-        DEPENDS
-            chewing-editor
-            ${h2m}
-    )
+    if (GZIP)
+        set(manpage ${PROJECT_BINARY_DIR}/chewing-editor.1.gz)
+
+        add_custom_command(
+            OUTPUT
+                ${manpage}
+            COMMAND ${HELP2MAN}
+                --include=${h2m}
+                --section=1
+                --no-info
+                --no-discard-stderr
+                ${PROJECT_BINARY_DIR}/chewing-editor
+                | ${GZIP} -c - > ${manpage}
```

Comment:
is that a pipe here? will that work on windows?

anyway, I think it's better to set a variable for `COMMAND` args, then append the additional arg when gzip is found.
reason behind this is to avoid dup. code.

the `set(manpage ...` is ok this way.